examples/provisioner/bottlerocket.yaml

```
@@ -23,7 +23,7 @@ spec:
     - deviceName: /dev/xvda
       ebs:
         volumeType: gp3
-        volumeSize: 2Gi
+        volumeSize: 2G # if using GPU, replace with 4G since the GPU bottlerocket AMI requires at least 4G
```

```suggestion
        volumeSize: 4Gi
```
Since 4Gi works consistently, I'd be fine just changing it to 4GI by default to reduce the chances people running into this.